src/DataValues/DataValue.php

```diff
@@ -10,7 +15,7 @@
  * @licence GNU GPL v2+
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
  */
-interface DataValue extends \Hashable, \Comparable, \Serializable, \Immutable {
+interface DataValue extends Hashable, Comparable, Serializable, Immutable {
```

Comment:
If it is a class in the top level NS and it is only used in the extends clause, then I find importing it a bit silly. To me it decreases readability more than it helps.